coral-common/src/main/java/com/linkedin/coral/common/transformers/SqlCallTransformer.java

```diff
@@ -96,4 +101,13 @@ protected RelDataType getRelDataType(SqlNode sqlNode) {
     }
     throw new RuntimeException("Failed to derive the RelDataType for SqlNode " + sqlNode);
   }
+
+  public static SqlOperator createSqlUDF(String functionName, SqlReturnTypeInference typeInference,
```

Comment:
I don't think it's suitable to add this helper method here, consider moving it to `com.linkedin.coral.common.calcite.CalciteUtil`. And `initSqlIdentifierWithFunctionNameList` parameter may not be necessary, we can create all with the list and test to see if it's needed.

Reply:
Done